src/constants/abacus.ts

```diff
@@ -278,6 +278,47 @@ export const ORDER_STATUS_STRINGS: Record<KotlinIrEnumValues<typeof AbacusOrderS
   [AbacusOrderStatus.untriggered.rawValue]: STRING_KEYS.UNTRIGGERED,
 };
 
+export const ORDER_TYPE_LABEL_MAPPING: Record<
```

Comment:
There is another label mapping object in `/constants/trade`, is there a clever way we could merge these? like 
```
[AbacusOrderType.___]: {
  ...
},
```

Reply:
ooh good point - updated label mapping object `TRADE_TYPE_STRINGS` -> `ORDER_TYPE_STRINGS` and removed abacus definition